app/services/search_query_parser.rb

```diff
@@ -0,0 +1,22 @@
+class SearchQueryParser
+  API_URL = ENV['TARIFF_QUERY_SEARCH_PARSER_URL']
+
+  def initialize
+    @client = Faraday.new(
+      url: API_URL,
+      headers: {'Content-Type' => 'application/json'}
```

Comment:
Setting the header can be handled automatically by using the `:json` Request middleware

Reply:
Done 👍